src/overlays/actors/ovl_En_Bji_01/z_en_bji_01.c

```diff
@@ -21,36 +44,405 @@ const ActorInit En_Bji_01_InitVars = {
+            break;
+    }
+}
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD328.asm")
+void EnBji01_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnBji01* this = THIS;
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD634.asm")
+    Collider_DestroyCylinder(globalCtx, &this->collider);
+}
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD6B0.asm")
+void EnBji01_Update(Actor* thisx, GlobalContext* globalCtx) {
+    static s16 sBlinkSequence[] = { 0, 1, 2, 1, 0, 0 };
+    EnBji01* this = THIS;
+    s32 pad;
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD6C0.asm")
+    this->actionFunc(this, globalCtx);
+    Actor_UpdateBgCheckInfo(globalCtx, (Actor*)this, 0.0f, 0.0f, 0.0f, 4U);
+    SkelAnime_FrameUpdateMatrix(&this->skelAnime);
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD70C.asm")
+    if (this->blinkTimer-- <= 0) {
+        if (--this->blinkSeqIndex < 0) {
+            this->blinkSeqIndex = 4;
+            this->blinkTimer = (Rand_ZeroOne() * 60.0f) + 20.0f;
+        } else {
+            this->eyeTexIndex = sBlinkSequence[this->blinkSeqIndex];
+        }
+    }
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD77C.asm")
+    Actor_SetHeight(&this->actor, 40.0f);
+    Collider_UpdateCylinder(&this->actor, &this->collider);
+    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+}
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/EnBji01_Init.asm")
+s32 EnBji01_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
+                             Actor* thisx) {
+    EnBji01* this = THIS;
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/EnBji01_Destroy.asm")
+    if ((limbIndex == 0) && ((globalCtx->gameplayFrames % 2) != 0)) {
+        *dList = NULL;
+    }
+    if (limbIndex == 0) {
+        rot->x = rot->x;
+        rot->y = rot->y;
+        rot->z = rot->z;
+    }
+    switch (limbIndex) {
+        case BJI_LIMB_TORSO:
+            rot->x += this->torsoXRotAdj;
+            rot->z += this->torsoZRotAdj;
+            break;
+        case BJI_LIMB_HEAD:
+            rot->x += this->headXRotAdj;
+            rot->z += this->headZRotAdj;
+            break;
+    }
+    return false;
+}
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/EnBji01_Update.asm")
+void EnBji01_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    static Vec3f D_809CDCC8 = { 1088.0f, 1200.0f, 0.0f };
+    EnBji01* this = THIS;
+    Vec3f sp20;
+    s32 temp_f4 = 0;
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CDA4C.asm")
+    if (limbIndex == 15) {
+        Math_Vec3f_Copy(&sp20, &D_809CDCC8);
```

Comment:
Now that you have the limb enum, you can use it to remove those magic numbers. 
```suggestion
    if (limbIndex == BJI_LIMB_HEAD) {
        Math_Vec3f_Copy(&sp20, &D_809CDCC8);
```